src/main/java/com/atlassian/db/replica/internal/aurora/AuroraJdbcUrl.java

```diff
@@ -1,12 +1,18 @@
 package com.atlassian.db.replica.internal.aurora;
 
-public class AuroraJdbcUrl {
-    private static final String PREFIX = "jdbc:postgresql://";
+import com.atlassian.db.replica.api.jdbc.JdbcProtocol;
+import com.atlassian.db.replica.api.jdbc.JdbcUrl;
 
+public class AuroraJdbcUrl extends JdbcUrl {
```

Comment:
Why not composition? Making JdbcUrl not final will make it hard to evolve.

Reply:
made JdbcUrl final and introduced a `toJdbcUrl` method to create a JdbcUrl using an AuroraJdbcUrl